libservices/mysqlservices_aix.def.in

```diff
@@ -0,0 +1,2 @@
+#! .
```

Comment:
actually, you could just check in a file with following content

libservices/mysqlservices_aix.def
```
#! .
   base64_service
    debug_sync_service
    encryption_scheme_service
    encryption_service
    json_service
    logger_service
    my_crypt_service
    my_md5_service
    my_print_error_service
    my_sha1_service
    my_sha2_service
    my_snprintf_service
    progress_report_service
    thd_alloc_service
    thd_autoinc_service
    thd_error_context_service
    thd_kill_statement_service
    thd_rnd_service
    thd_specifics_service
    thd_timezone_service
    thd_wait_service
    wsrep_service
```
I mean , rather than generate it via cmake, or am I missing anything?  Then you'd  use 

"-Wl,-bE:${CMAKE_SOURCE_DIR}/libservices/mysqlservices_aix.def"
instead of 

"-Wl,-bE:${CMAKE_BINARY_DIR}/libservices/mysqlservices_aix.def"

in the single place it is used, in macro MYSQL_ADD_PLUGIN.

there are cases, where generating makes sense but here, it is a single-purpose, workaround for "GCC on AIX does not have anything similar to attribute visibility(default) or declspec(dllexport)". really, nothing? no -fvisibility? no -qvisibility?

Reply:
I can create it directly. I generate it to be consistant. But, yes, it is a workaround.

And, non, there is no -q/fvisibility for GCC on AIX. Only export and import file. I am really sorry. It will be more simple, else.